agr_java_core/src/main/java/org/alliancegenome/es/util/ProcessDisplayHelper.java

```diff
@@ -125,6 +133,10 @@ public void finishProcess() {
 	}
 	
 	public void finishProcess(String data) {
+		// do not do display processing when only few records in total are loaded.
+		if(totalSize < MINIMUM_TOTAL)
```

Comment:
@cmpich this break all the other cases where totalSize is not set.

Reply:
Removed.